src/main/scala/nl.knaw.dans.easy.deposit/servlets/DepositServlet.scala

```diff
@@ -58,12 +58,15 @@ class DepositServlet(app: EasyDepositApiApp)
    * Anyhow, the implicit `user.id` is not safe outside a for-comprehension as (in theory) it can be null.
    */
 
+  before() {
+    contentType = "application/json;charset=UTF-8"
```

Comment:
* it logs

      INFO  GET returned status=500; ..., Content-Type -> [text/plain;charset=UTF-8]]; actionHeaders=[content-type -> text/plain;charset=UTF-8]

  for unit test https://github.com/DANS-KNAW/easy-deposit-api/blob/5b791bc92de1041cc95b80446d69ea7a41efe6bd/src/test/scala/nl.knaw.dans.easy.deposit/servlets/IntegrationSpec.scala#L75
* it will provide content headers for responses with an empty body. Do we want that?

Reply:
See also cover message for planned manual test